src/main/java/nextstep/ladder/domain/Person.java

```diff
@@ -0,0 +1,17 @@
+package nextstep.ladder.domain;
+
+public class Person {
+    private final String name;
+
+    public Person(String name) {
+        if (name.length() > 5) {
```

Comment:
```suggestion
        if (name.length() > NAME_LENGTH_LIMIT) {
```

매직 넘버는 상수로 분리해서 가독성을 높힐 수 있습니다. 
물론, 재사용성이 필요없고, 그 숫자(혹은 문자) 자체로 상수보다 의미를 줄 수 있을 경우라면 꼭 분리하지 않아도 되니 잘 선택해서 사용해주시면 됩니다 ㅎ